english/american-sniper/american-sniper.md

```diff
@@ -0,0 +1,3 @@
+The story in the movie is good, heart touching and reflects courage. However, it is far away from what truly happened there...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.